### PR TITLE
Fix inserting large spatial datatypes using setObject or setGeometry/setGeography

### PR DESCRIPTION
**Pull request by peterbae**
Fixes issue #1270.

When the driver is inserting byte[] data into a column in the server's table, the driver changes the JdbcType of the column (effectively, we change what we send the data as in the INSERT query) if the byte array is larger than the 8000. For example, this means that if the driver was going to insert the data as geometry, it was instead sending varbinary(max) if the byte array that represents the geometry/geography data was larger than 8000. Excluding geometry/geography from being typecast into varbinary(max) solves this issue (geometry/geography data types don't have a size limit).

### CODE REVIEW

File: src/test/java/com/microsoft/sqlserver/jdbc/datatypes/SQLServerSpatialDatatypeTest.java [hunk 1]
```diff
@@ -1142,6 +1142,63 @@ public void testWrongtype() throws SQLException {
...
+        beforeEachSetup();
+
+        Geometry geomWKT = Geometry.STGeomFromText(wktLargeGeometry, 0);
```
Comment (mprins) [inline src/test/java/com/microsoft/sqlserver/jdbc/datatypes/SQLServerSpatialDatatypeTest.java:1157]: I don't it matters much, but these coordinates are in SRID 28992

**Comment by ulvii** [comment]
[PR1337.zip](https://github.com/microsoft/mssql-jdbc/files/4652810/PR1337.zip)
@mprins 
I attached the binaries to this PR, would you give them a try and let us know if your issues are resolved?

**Comment by mprins** [comment]
@ulvii Thanks, I can/will give this a try at the end of the week at the earliest

File: src/test/java/com/microsoft/sqlserver/jdbc/datatypes/SQLServerSpatialDatatypeTest.java (hunk 1 above)
Comment (peterbae) [reply src/test/java/com/microsoft/sqlserver/jdbc/datatypes/SQLServerSpatialDatatypeTest.java:1157]: Thanks. For the purpose of this test, it shouldn't make a difference.

**Comment by peterbae** [comment]
Hi @mprins, did you have a chance to test out the changes?

**Comment by mprins** [comment]
I've ran 2 projects with this updated driver and all tests appear to be passing, including that reported in #1270

**Pull request merged**